.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug
+description: Report a bug
+title: "[Bug] "
```

Comment:
I think we agreed to drop these.

Reply:
Fixed in [refactor: Drops all title pre-fixes](https://github.com/HHS/simpler-grants-gov/pull/2369/commits/ee23fa1a73749925203c3351a58a0659a98cdc33)